src/encoded/static/components/globals.js

```diff
@@ -181,7 +181,7 @@ export function isHashPartOfHref(href, parts = null){
     if (!parts) parts = url.parse(href);
 
     // Edit Pages
-    if (parts.hash === '#!edit' || parts.hash === '#!create') return true;
+    if (parts.hash === '#!edit' || parts.hash === '#!create' || parts.hash === '#!add') return true;
```

Comment:
Allows us to navigate to `/search/?type=MyItem#!add` and keep the `#!add` in href to launch straight into submission view (e.g. from a different URL, such as health page).